code/game/machinery/kitchen/cooking_machines/container.dm

```diff
@@ -8,6 +8,10 @@
 	var/max_space = 20//Maximum sum of w-classes of foods in this container at once
 	var/max_reagents = 80//Maximum units of reagents
 
+	var/list/insertable = list(/obj/item/weapon/reagent_containers/food/snacks,
+	/obj/item/weapon/holder,
+	/obj/item/weapon/paper)
```

Comment:
If any recipes are added with unusual non-food ingredients in future, this list will need extended